.ado/templates/e2e-testing-win32.yml

```diff
@@ -25,6 +25,12 @@ steps:
     condition: succeeded()
     displayName: 'Create success build variable'
 
+  - script: |
+      yarn appium driver install --source=npm appium-windows-driver
```

Comment:
Curious about this - why do we need to add it now?

We have the "appium-windows-driver" in apps/win32 package.json which should download the driver to apps/win32/node_modules

Reply:
You still need to tell appium to install the driver, it doesn't automatically pick up the driver. I wasn't able to find a way to tell appium "use the already downloaded folder" without running into problems.

Reply:
Interesting, that seems like a weird regression with Appium then. Previously, I found it would automatically search in the current node_modules folder.

If your running it locally, does it also require you to run that driver install command?

Reply:
Yeah, it refused to run unless I told it to install the driver.
It does have some behavior to automatically find the driver, but only using npx. I'm not sure if it's a appium regression or if it's a difference in behavior due to changing our node version.